src/common/types/core.js

```diff
@@ -77,6 +77,8 @@ type Access = typeof ACCESS_COLLAB | typeof ACCESS_COMPANY | typeof ACCESS_OPEN;
 
 type InlineNoticeType = 'warning' | 'error' | 'success' | 'info' | 'generic';
 
+type NotificationType = 'default' | 'info' | 'warn' | 'error';
```

Comment:
itd be nice to unify `InlineNoticeType` and `NotificationType` so we don't have these nearly identical types. possible in your change?